modules/migration-enabling-pv-resizing-dvm.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration-toolkit-for-containers/advanced-migration-options-mtc.adoc
+
+[id="migration-enabling-pv-resizing-dvm_{context}"]
+= Enabling persistent volume resizing for direct volume migration
+
+You can enable persistent volume (PV) resizing for direct volume migration to avoid running out of disk space on the destination cluster.
+
+When the disk usage of a PV reaches a configured level, the `MigrationController` custom resource (CR) compares the requested storage capacity of a persistent volume claim (PVC) to its actual provisioned capacity. Then, it calculates the space required on the destination cluster.
+
+A `pv_resizing_threshold` parameter determines when PV resizing is used. The default threshold is `3%`. This means that PV resizing occurs when the disk usage of a PV is more than `97%`. You can increase this threshold so that PV resizing occurs at a lower disk usage level.
+
+PVC capacity is calculated according to the following criteria:
+
+* If the requested storage capacity (`spec.resources.requests.storage`) of the PVC is not equal to its actual provisioned capacity (`status.capacity.storage`), the greater value is used.
+* If a PV is provisioned through a PVC and then subsequently changed so that its PV and PVC capacities no longer match, the greater value is used.
+
+.Prerequisites
+
+* The PVCs must be attached to one or more running pods so that the `MigrationController` CR can execute commands.
+
+.Procedure
+
+. Log in to the host cluster.
+. Enable PV resizing by patching the `MigrationController` CR:
++
+[source,terminal]
+----
+$ oc patch migrationcontroller migration-controller -p '{"spec":{"enable_dvm_pv_resizing":true}}' \ <1>
+  --type='merge' -n openshift-migration
+----
+<1> Set the value to `false` to disable PV resizing.
+
+. Optional: Update the `pv_resizing_threshold` parameter to increase the threshold:
++
+[source,terminal]
+----
+$ oc patch migrationcontroller migration-controller -p '{"spec":{"pv_resizing_threshold":41}}' \ <1>
+  --type='merge' -n openshift-migration
+----
+<1> The default value is `3`.
++
```

Comment:
Should you combine these two steps so that the use only needs to patch once to enable and increase the threshold? 

```
----
$ oc patch migrationcontroller migration-controller -p '{"spec":{"enable_dvm_pv_resizing":true}{"pv_resizing_threshold":41}}' \ <1><2>
  --type='merge' -n openshift-migration
----
<1> Set the value to `false` to disable PV resizing.
 <2> Optional: Update the `pv_resizing_threshold` parameter to increase the threshold. The default value is `3`.

```
Not sure on the exact formatting of the command or call-outs (maybe that's why you did two steps?)